source/conf.py

```diff
@@ -7,6 +8,10 @@
 # sys.path.append(os.path.abspath('.'))
 
 # -- General configuration -----------------------------------------------------
+if not os.path.exists('../invest-sample-data'):
+    subprocess.run(['make', '-C', '..', 'get_sampledata'])
+if not os.path.exists('invest-sample-data/pollination/landcover_biophysical_table_modified.csv'):
```

Comment:
This isn't very neat but I think for now it is okay, we will probably not keep building on GHA and RTD simultaneously for a long time.

Reply:
So this `conf.py` runs during both workflows, but with GHA we will have already done this prep in earlier workflow steps? 

Seems like a fine solution to me.

Reply:
Yea, this is just for RTD